<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoteWindow</class>
 <widget class="QMainWindow" name="RemoteWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1254</width>
    <height>628</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>.AppleSystemUIFont</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>远程控制终端</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonGoHead">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>170</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>前进</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTurnLeft">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>240</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>左转</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGoBack">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>300</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>后退</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTurnRight">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>240</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>右转</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditLinkIp">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>450</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>nps.areay7.cn</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLink">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>450</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditLinkPort">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>500</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>8098</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelIpText">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>450</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPortText">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>500</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>端口</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditInfomation">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>40</y>
      <width>221</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1254</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
